### DISCUSSION TITLE
Web search style optimization

### DISCUSSION THREAD

**Discussion by cloudmoonocus** [mdn/mdn-community#139] [Platform] [answered]
Can you please optimize the search style? When I'm searching, I can't identify exactly if I'm searching. Maybe there should be a bottom color and or something that promt me.

**Comment by schalkneethling** [comment]
> Can you please optimize the search style? When I'm searching, I can't identify exactly if I'm searching. Maybe there should be a bottom color and or something that promt me.

Hey there, would you mind expanding on this somewhat? Perhaps post some screenshots or a short video? Thank you.

**Reply by cloudmoonocus** [reply, marked as answer]
Sorry, when I was going to take a screenshot just now, I found out that it was my browser theme issue. It's all right now, thank you!

**Reply by schalkneethling** [reply]
Thank you.
